operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 * [ENHANCEMENT] Set `nginx` and `gateway` Nginx read timeout (`proxy_read_timeout`) to 300 seconds (increase from default 60 seconds), so that it doesn't interfere with the querier's default 120 seconds timeout (`mimir.structuredConfig.querier.timeout`). #4924
 * [ENHANCEMENT] Update the `rollout-operator` subchart to `0.5.0`. #4930
+* [ENHANCEMENT] Store-gateway: set `GOMEMLIMIT` to the memory request value. #4971
```

Comment:
Can you give a bit more detail for what that means for a user? It's effectively trading memory usage for CPU, so if someone sees increased CPU after an upgrade they can infer to increase the memory request.